ratelimiter/src/main/java/com/flipkart/varadhi/qos/entity/LoadPrediction.java

```diff
@@ -0,0 +1,9 @@
+package com.flipkart.varadhi.qos.entity;
+
+import java.util.Deque;
+import java.util.List;
+import java.util.Map;
+
+public interface LoadPrediction {
+    List<TopicLoadInfo> predictLoad(Map<String, Deque<TopicLoadInfo>> records);
```

Comment:
documentation?

Why does it return List<TopicLoadInfo>?

Map<String, Deque<TopicLoadInfo>> records is also weird type for it to receive. What is String here?

Reply:
Added docs, and make client history to handle the data. Now it doesn't make sense to send any information through params